test/multiverse/suites/mongo/Envfile

```diff
@@ -35,8 +31,11 @@ def mongo_versions
   versions += MONGO_VERSIONS_RUBY26 if ruby_gte?('2.6.0')
   return versions if RUBY_PLATFORM.eql?('java')
 
-  versions += MONGO_VERSIONS_RUBY25 if ruby_gte?('2.5.0')
-  versions += MONGO_VERSIONS_ALL_RUBIES
+  if ruby_gte?('2.5.0')
```

Comment:
What does `gte` mean? I'm not familiar with that abbreviation.

Reply:
`gte` = "greater than or equal to"

In many places we simply compare strings mathematically and benefit from Ruby's minor version never hitting double digits (`'3.10.0'` would be considered smaller than `'3.2.0'`). The `gte` helper method used `Gem::Version` comparisons instead.